src/Tools/Shared/TestHelpers/TemporaryCSharpProject.cs

```diff
@@ -1,17 +1,17 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Text;
 
-namespace Microsoft.DotNet.Watcher.Tools.Tests
+namespace Microsoft.DotNet.Tools
 {
     public class TemporaryCSharpProject
     {
         private const string Template =
- @"<Project Sdk=""Microsoft.NET.Sdk"">
+ @"<Project Sdk=""Microsoft.NET.Sdk.Web"">
```

Comment:
We actually run the Temporary project in one case as a smoke test, without making this a web SDK it fails to compile because of missing namespaces. It shouldn't really affect any of the other tests because none of them build their projects.